CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Getting Started
+
+Install Node.js if you haven't already and run:
+`npm install`
+
+Install **Fancybox** via the command:
```

Comment:
This file is out of date now, please update to reflect the fact that you only need to do `npm install`